helm/default-apps-gcp/templates/apps.yaml

```diff
@@ -6,6 +6,10 @@ kind: App
 metadata:
   annotations:
     chart-operator.giantswarm.io/force-helm-upgrade: "{{ .forceUpgrade }}"
+    {{- if .dependsOn }}
+    # App is deployed in the org- namespace so the secret name is prefixed by the cluster-id
```

Comment:
Same as on the other PR, I think the comment is wrong.

Reply:
I think you can remove this comment tbh

Reply:
Or say that cluster operator will ensure apps that this one depends on will be installed first